io/plugins/eu.esdihumboldt.hale.io.xls/src/eu/esdihumboldt/hale/io/xls/writer/XLSInstanceWriter.java

```diff
@@ -120,14 +124,15 @@ else if (getContentType().getId().equals("eu.esdihumboldt.hale.io.xls.xlsx")) {
 		Row headerRow = sheet.createRow(0);
 		int rowNum = 1;
 		Row row = sheet.createRow(rowNum++);
-		writeRow(row, super.getPropertyMap(instance, headerRowStrings, solveNestedProperties));
+		writeRow(row,
+				super.getPropertyMap(instance, headerRowStrings, solveNestedProperties, useSchema));
 
 		while (instanceIterator.hasNext()) {
 			Instance nextInst = instanceIterator.next();
 			if (nextInst.getDefinition().equals(definition)) {
 				row = sheet.createRow(rowNum++);
-				writeRow(row,
-						super.getPropertyMap(nextInst, headerRowStrings, solveNestedProperties));
+				writeRow(row, super.getPropertyMap(nextInst, headerRowStrings,
```

Comment:
The method declaration is
```
protected Map<String, Object> getPropertyMap(Instance instance, List<String> headerRow,
			boolean useSchema, boolean solveNestedProperties)
```

So I think you pass the arguments the wrong way.

Reply:
sorry!